components/globe-component/globe-component.js

```diff
@@ -43,6 +42,13 @@ export default {
 
     this.controls = new OrbitControls(this.camera, this.globeContainerElement)
     this.controls.enablePan = false
+    this.controls.minDistance = 6
+    this.controls.maxDistance = 50
+
+    const that = this
+    this.controls.addEventListener('change', function () {
+      that.$store.commit('enableGlobeAutoRotation', false)
+    })
```

Comment:
Do you need to switch this? or can you also use an arrow function that would maintain the `this` binding